docs/getting-started/development-environment.rst

```diff
@@ -9,10 +9,161 @@
 
 Developer environment guide
 ===========================
-You can save a lot of time and frustrations by spending some time setting up
-your development environment. We have primarily adopted existing community
-style guides, and in most cases the formatting and checking can be fully
-automated by your editor.
+
+The following is a guide to help you prepare your system for developing with
+Invenio. A proper development environment setup can save a lot of time and
+frustrations. Note that the following guide, is not meant as a guide for
+setting up servers.
+
+System setup
+------------
+We support development on macOS and Linux-based systems (development on Windows
+is not supported).
+
+macOS
+~~~~~
+We recommend that you install libraries and tools using the
+`Homebrew <https://brew.sh>`_ package manager. Homebrew allows you to install
+two types of packages - cask and normal.
+
+**Cask packages**
+
+Cask packages are usually UI applications that ends up en your Applications
+folder. You can install cask packages using the command:
+
+.. code-block:: console
+
+    $ brew cask install <packages>
+
+Here are some recommended cask packages:
+
+.. code-block:: console
+
+    discord            # app that we use for chat among developers
+    docker             # docker setup for macOS
+    google-chrome      # needed for some end-to-end tests
+    iterm2             # a better terminal than Terminal.app
+    postico            # UI for PostgreSQL
```

Comment:
Not to polute, so it's ok as it is. I use `DB browser for SQLite` and it works pretty fine for sqlite. https://sqlitebrowser.org

Reply:
Let's add it :-) Idea is that people can select what they want. Probably I should add only docker is really required
``db-browser-for-sqlite``